From 7645dd9cc0f047e6e41256401ae26ecc2a00b373 Mon Sep 17 00:00:00 2001
From: Bob Beckett <bob.beckett@collabora.com>
Date: Tue, 14 Nov 2023 11:22:47 +0100
Subject: [PATCH 28/29] panfrost: Add an entry for panthor in the
 renderonly_drivers[] array

Same as panfrost, but the kernel driver has a different name.
---
 src/gallium/winsys/kmsro/drm/kmsro_drm_winsys.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/gallium/winsys/kmsro/drm/kmsro_drm_winsys.c b/src/gallium/winsys/kmsro/drm/kmsro_drm_winsys.c
index 3d14bce1ce7..259e876dc8b 100644
--- a/src/gallium/winsys/kmsro/drm/kmsro_drm_winsys.c
+++ b/src/gallium/winsys/kmsro/drm/kmsro_drm_winsys.c
@@ -101,7 +101,8 @@ struct pipe_screen *kmsro_drm_screen_create(int kms_fd,
       ro->create_for_resource = renderonly_create_kms_dumb_buffer_for_resource;
       screen = lima_drm_screen_create_renderonly(ro->gpu_fd, ro, config);
 #endif
-   } else if (strcmp(render_dev_name, "panfrost") == 0) {
+   } else if (strcmp(render_dev_name, "panfrost") == 0 ||
+              strcmp(render_dev_name, "panthor") == 0) {
 #if defined(GALLIUM_PANFROST)
       ro->create_for_resource = panfrost_create_kms_dumb_buffer_for_resource;
       screen = panfrost_drm_screen_create_renderonly(ro->gpu_fd, ro, config);
-- 
2.43.0

